env0/utils_test.go

```diff
@@ -0,0 +1,75 @@
+package env0
+
+import (
+	"testing"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestReadResourceData(t *testing.T) {
```

Comment:
Since you're actually testing the `module` resource scenario, could you perhaps rename this test function to cope with that scenario?
I assume any future user of this util might need to add test for any other resource types that support new attribute types (such as, new array types for e.g)

Reply:
https://github.com/env0/terraform-provider-env0/pull/279/commits/a59b4bbbb4c4f0f861546b4848b2c55e5040adbb